cloudsql-mysql-plugin/src/e2e-test/java/io/cdap/plugin/CloudMySql/BQValidation.java

```diff
@@ -0,0 +1,225 @@
+                        break;
+
+                    case Types.SMALLINT:
+                    case Types.INTEGER:
+                    case Types.TINYINT:
+                        Integer sourceTinyInt = rsSource.getInt(currentColumnCount);
+                        Integer targetTinyInt = Integer.parseInt(bigQueryData.get(jsonObjectIdx).get(columnName).getAsString());
+                        Assert.assertTrue("Different values found for column : %s",
+                                String.valueOf(sourceTinyInt).equals(String.valueOf(targetTinyInt)));
+                        break;
+
+                    case Types.REAL:
+                        Float sourceFloat = rsSource.getFloat(currentColumnCount);
+                        Float targetFloat = Float.parseFloat(bigQueryData.get(jsonObjectIdx).get(columnName).getAsString());
+                        Assert.assertTrue("Different values found for column : %s",
+                                String.valueOf(sourceFloat).equals(String.valueOf(targetFloat)));
+                        break;
+
+                    case Types.DOUBLE:
+                        Double sourceDouble= rsSource.getDouble(currentColumnCount);
+                        Double targetDouble = Double.parseDouble(bigQueryData.get(jsonObjectIdx).get(columnName).getAsString());
+                        Assert.assertTrue("Different values found for column : %s",
+                                String.valueOf(sourceDouble).equals(String.valueOf(targetDouble)));
+                        break;
+
+                    case Types.DATE:
+                        Date sourceDate = rsSource.getDate(currentColumnCount);
+                        Date targetDate = java.sql.Date.valueOf(bigQueryData.get(jsonObjectIdx).get(columnName).getAsString());
+                        Assert.assertTrue("Different values found for column : %s",
+                                String.valueOf(sourceDate).equals(String.valueOf(targetDate)));
+                        break;
+
+                    case Types.TIME:
+                        Time sourceTime= rsSource.getTime(currentColumnCount);
+                        Time targetTime = Time.valueOf(bigQueryData.get(jsonObjectIdx).get(columnName).getAsString());
+                        Assert.assertTrue("Different values found for column : %s",
+                                String.valueOf(sourceTime).equals(String.valueOf(targetTime)));
+                        break;
+
+                    case Types.DECIMAL:
+                        org.apache.spark.sql.types.Decimal sourceDecimal = org.apache.spark.sql.types.Decimal.fromDecimal(rsSource.getBigDecimal(currentColumnCount));
+                        org.apache.spark.sql.types.Decimal targetDecimal = org.apache.spark.sql.types.Decimal.fromDecimal(bigQueryData.get(jsonObjectIdx).get(columnName).getAsBigDecimal());
+                        Assert.assertEquals("Different values found for column : %s", sourceDecimal, targetDecimal);
```

Comment:
Don't directly import it here.

Reply:
done
commit link- b03766131af7a687ad66f495a6eac551f74c15d8